core/runtime/src/main/java/io/quarkus/runtime/configuration/ConverterSupport.java

```diff
@@ -0,0 +1,157 @@
+package io.quarkus.runtime.configuration;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.ServiceLoader;
+import java.util.function.Consumer;
+
+import javax.annotation.Priority;
+
+import org.eclipse.microprofile.config.spi.ConfigBuilder;
+import org.eclipse.microprofile.config.spi.Converter;
+import org.jboss.logging.Logger;
+
+import io.smallrye.config.SmallRyeConfigBuilder;
+
+/**
+ * This small utility class is a tool which helps populating {@link SmallRyeConfigBuilder} with
+ * {@link Converter} implementations loaded from {@link ServiceLoader}.
+ *
+ * @author Bartosz Firyn (sarxos)
```

Comment:
Could you please get rid of all the `@author` tags in the PR? We decided we do not want them.

Reply:
Sure. My eclipse is generates them automatically for all of my classes.